src/pages/api/breadcrumbs.ts

```diff
@@ -4,11 +4,13 @@ import { NextApiRequest, NextApiResponse } from 'next';
 import { ZetkinViewFolder } from 'features/views/components/types';
 
 interface LabeledBreadcrumbElement {
+  folderId?: string;
```

Comment:
I feel like this is a little bit weird, to add potential folders to all elements, but it clearly works!

Reply:
It felt okay to assume more types could have folder structures in the future, and avoid having another specific interface.